Gemfile

```diff
@@ -84,6 +84,7 @@ group :development, :test do
   gem "pry"
```

Comment:
Can probably remove 'pry' gem if your bringing in `pry-rails` as it supersedes it.

Also is anyone using pry-remote or pry-byebug?